tests/strategies/test_deepspeed_strategy.py

```diff
@@ -1322,3 +1322,17 @@ def test_deepspeed_with_bfloat16_precision(tmpdir):
     assert trainer.strategy.precision_plugin.precision == "bf16"
     assert trainer.strategy.config["zero_optimization"]["stage"] == 3
     assert model.layer.weight.dtype == torch.bfloat16
+
+
+@RunIf(standalone=True, deepspeed=True)
```

Comment:
I might be wrong, but I don't think we need this to be of standalone. Let's try and see if it works in GPU CI. If it doesn't we can add enable it back :]
```suggestion
@RunIf(deepspeed=True)
```